inf-clojure.el

```diff
@@ -261,12 +267,12 @@ whichever process buffer you want to use.")
 
 (put 'inf-clojure-mode 'mode-class 'special)
 
-(defcustom inf-clojure-repl-default-flavor 'clojure
+(defcustom inf-clojure-repl-default-flavor nil
   "Symbol to define your default REPL flavor.
 The default flavor is clojure, lumo is the other supported
 one."
   :type '(choice (const :tag "Lumo" lumo)
-                 (other :tag "Clojure" clojure))
+                 (other :tag "Project driven" nil))
```

Comment:
This tag here clarifies that in case of missing flavor the project type will drive